tests/integration/distribution/keeper/msg_server_test.go

```diff
@@ -206,3 +219,49 @@ func (s *KeeperTestSuite) TestCommunityPoolSpend() {
 		})
 	}
 }
+
+func TestDepositValidatorRewardsPool(t *testing.T) {
```

Comment:
can this be converted to tabular format tests (ex: look at `TestMsgUpdateParams` at L21)